gemfirexd/shared/build.gradle

```diff
@@ -32,6 +32,7 @@ task createVersionPropertiesFile(dependsOn: 'processResources') {
     def props = [
       'Product-Name'      : PRODUCT_NAME,
       'Product-Version'   : version,
+      'Enterprise-Edition': true,
```

Comment:
Yes I will work on making it configurable, right now it was just a placeholder to see if the name is OK. thanks!